arm-sql/2014-04-01/swagger/metrics.json

```diff
@@ -0,0 +1,451 @@
+            "schema": {
+              "$ref": "#/definitions/MetricDefinitionListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MetricValue": {
+      "properties": {
+        "count": {
+          "readOnly": true,
+          "type": "number",
+          "format": "int32",
+          "description": "The number of values for the metric."
+        },
+        "average": {
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "description": "The average value of the metric."
+        },
+        "maximum": {
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "description": "The max value of the metric."
+        },
+        "minimum": {
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "description": "The min value of the metric."
+        },
+        "timestamp": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The metric timestamp (ISO-8601 format)."
+        },
+        "total": {
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "description": "The total value of the metric."
+        }
+      },
+      "description": "Represents database metrics.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SubResource"
+        }
+      ]
+    },
+    "MetricName": {
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The name of the database metric."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "The friendly name of the database metric."
+        }
+      },
+      "description": "A database metric name."
+    },
+    "Metric": {
+      "properties": {
+        "startTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time for the metric (ISO-8601 format)."
+        },
+        "endTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The end time for the metric (ISO-8601 format)."
+        },
+        "timeGrain": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The time step to be used to summarize the metric values."
+        },
+        "unit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unit of the metric.",
+          "enum": ["count", "bytes", "seconds", "percent", "countPerSecond", "bytesPerSecond"],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "UnitType"
+          }
+        },
+        "name": {
+          "$ref": "#/definitions/MetricName",
```

Comment:
My bad.  I thought you meant tagging the properties in the $ref'd definitions "required," not the $ref themselves